src/test/kotlin/com/fasterxml/jackson/module/kotlin/test/ParameterNameTests.kt

```diff
@@ -199,8 +199,8 @@ class TestJacksonWithKotlin {
 
     private class StateObjectWithFactoryOnNamedCompanion private constructor (override val name: String, override val age: Int, override val primaryAddress: String, override val wrongName: Boolean, override val createdDt: Date) : TestFields {
         var factoryUsed: Boolean = false
-        companion object Named {
-            @JvmStatic @JsonCreator fun create(@JsonProperty("name") nameThing: String, @JsonProperty("age") age: Int, @JsonProperty("primaryAddress") primaryAddress: String, @JsonProperty("renamed") wrongName: Boolean, @JsonProperty("createdDt") createdDt: Date): StateObjectWithFactoryOnNamedCompanion {
+        private companion object Named {
+            @JvmStatic @JsonCreator private fun create(@JsonProperty("name") nameThing: String, @JsonProperty("age") age: Int, @JsonProperty("primaryAddress") primaryAddress: String, @JsonProperty("renamed") wrongName: Boolean, @JsonProperty("createdDt") createdDt: Date): StateObjectWithFactoryOnNamedCompanion {
```

Comment:
Would you split this over lines (if you're using IntelliJ, ⌥⏎ then select `Put parameters on separate lines`):

<img width="405" alt="image" src="https://user-images.githubusercontent.com/23986/148051738-c84c2b36-a2d2-43b1-aef0-7a12297dafed.png">

Reply:
All lines that were exceeding 120 characters in length depending on the parameter were modified to break lines.
b19c44f